dashboard/app/helpers/application_helper.rb

```diff
@@ -48,9 +48,15 @@ def check_mark_html
     image_tag(image_url('white-checkmark.png'))
   end
 
-  def activity_css_class(result, submitted=false)
+  def activity_css_class(user_level)
     # For definitions of the result values, see /app/src/constants.js.
-    if submitted
+    result = user_level.try(:best_result)
+
+    if result == Activity::REVIEW_REJECTED_RESULT
```

Comment:
Can a level have state rejected/accepted without submitted? If not, then maybe it should be

if user_level.try(:submitted)
    if (user_level.try(:rejected))
       rejected
    elseif (user_level.try(:accepted))
        accepted
    else
        submitted
else
   'not_tried'